src/transforms/kotlin/svgToDrawable.ts

```diff
@@ -35,7 +35,7 @@ export default {
 
     // Snake case and replace `icon` with `ic` as this is the convention on Android
     // and on Material
-    const imageId = _.snakeCase(token.name.replace("icon", "ic"));
+    const imageId = _.snakeCase(token.name.replace("icon", "ic_compound_"));
```

Comment:
Maybe `ic_cpd_` would be a bit nicer to use? `cpd` is the namespace prefix that we're using for Web, though if you find `ic_compound_` clearer that's completely valid

Reply:
I think `ic_compound_` is easier to understand for newcomers than `ic_cpd_`, and it's not that much longer. Thanks for the suggestion though!